packages/renderer/src/lib/help/HelpItems.ts

```diff
@@ -16,16 +16,6 @@
  * SPDX-License-Identifier: Apache-2.0
  ***********************************************************************/
 
-import { faDiscord, faSlack } from '@fortawesome/free-brands-svg-icons';
-import { faComment } from '@fortawesome/free-regular-svg-icons';
-import {
-  faExclamationTriangle,
-  faExternalLinkAlt,
-  faLightbulb,
-  faListUl,
-  type IconDefinition,
-} from '@fortawesome/free-solid-svg-icons';
-
 import { homepage, repository } from '../../../../../package.json';
```

Comment:
I find it interesting that we are doing a major refactor because we don't want to import the `product.json` but we are already importing the `package.json`